data/bio.js

```diff
@@ -0,0 +1,49 @@
+import { defineMessages } from 'react-intl'
+
+export const bios = defineMessages({
```

Comment:
As the about page is not localisable, I would not translate those bio either.

Reply:
Sounds good. Will remove.